CONTRIBUTING.md

```diff
@@ -0,0 +1,325 @@
+# Contributing Guide
+
+## For Contributors
+
+Contributions are always welcome!
+If you have discovered a bug or have a feature request, [please create an issue on GitHub](https://github.com/kintone-labs/kintone-ui-component/issues/new/choose).  
+Pull requests are also welcome when you find trivial bugs or typos.
+
+We use English for all commit messages, code comments, issues, pull requests.
+
+### Setup
+This repository is a monorepo using npm.
+```sh
+% cd kintone-ui-component
+% npm ci
+```
+
+### Build
+The project should be Setup before Build.
+```sh
+% cd kintone-ui-component
+% npm run build:umd_prod
+```
+
+### Development
+The project should be Setup before Development.
+
+```sh
+% cd kintone-ui-component
+% npm run build:umd_dev
+```
```

Comment:
I think we can summarize like this.

```suggestion
### Build
The project should be Setup before Build.

#### UMD (prod mode)
```sh
% cd kintone-ui-component
% npm run build:umd_prod
```

#### UMD (dev mode)
```sh
% cd kintone-ui-component
% npm run build:umd_dev
```

#### ESM
```sh
% cd kintone-ui-component
% npm run build:esm
```
```